index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Benson Portfolio</title>
+</head>
+<body>
+    <header id="header">
+        <section id="headertoolbar">
+            <div id="headerlogo">My Logo</div>
+            <nav class="navheader">
+                <button id="headermenubtn" type="button"><img src="/images/Union.png" alt="dropdown menu"></button>
+                <div class="desktop">
+                    <a href="#works">Portfolio</a>
+                    <a href="#about">About</a>
+                    <a href="#contact">Contact</a>
+                </div>
```

Comment:
I suggest you use the `ul` tag instead of the `div` tag to list the nav items and note each link (`a` tag) should be inside a `li` tag. in that case you'll have:

``` 
<ul>
  <li><a href="#">item 1</a></li>
  <li><a href="#">item 2</a></li>
  <li><a href="#">item 3</a></li>
</ul>
```

Reply:
- Grouped header toolbar links in `ul` tags and added relevant styling class to `styles.css` file.